website/docs/release_notes.md

```diff
@@ -5,6 +5,140 @@ sidebar_position: 99
+```
+
+Added in [#1336](https://github.com/VirtusLab/scala-cli/pull/1336) by [@alexarchambault](https://github.com/alexarchambault)
+
+## `-d`, `-classpath` and `compile` sub-command's `--output` options changes
+
+To be backward compatible with `scala` command a lot of changes have been made to the following options:
+* The `compile`'s  `--output` option has been renamed to `--compilation-output`. This option is now also available from the `run` and `package` sub-commands:
+
+```
+▶ scala-cli compile Hello.scala --compilation-output out
+▶ scala-cli --main-class Hello -classpath out
+Hello
+```
+
+* The `-d` option is no longer an alias for `--dependency`, but for `--compilation-output`. 
+  * `-O -d -O path/to/compilation/output` now defaults to `-d path/to/compilation/output`.
+
+```
+▶ scala-cli compile Hello.scala -d out
+▶ scala-cli --main-class Hello -classpath out
+Hello
+```
+
+* The old `--classpath` option has been renamed to `--print-classpath`.
+  *  `--classpath`, `--class-path` and `-classpath` options are now aliases for the `--extra jars` option.
+  * `-O -classpath -O path/to/classpath` now defaults to `--extra-jars path/to/classpath`.
+
+```
+▶ scala-cli compile --print-classpath Hello.scala
+# /work/.scala/project-cef76d561e/classes:~/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.14/scala-library-2.12.14.jar:~/Library/Caches/ScalaCli/local-repo/0.1.0/org.virtuslab.scala-cli/runner_2.12/0.0.1-SNAPSHOT/jars/runner_2.12.jar:~/Library/Caches/ScalaCli/local-repo/0.1.0/org.virtuslab.scala-cli/stubs/0.0.1-SNAPSHOT/jars/stubs.jar
+```
+
+Added in [#1340](https://github.com/VirtusLab/scala-cli/pull/1340) by [@Gedochao](https://github.com/Gedochao)
+
+## Make inputs optional when `-classpath` and `--main-class` are passed
+
+Following changes have been made to be backward compatible with `scala` command:
+* Passing `--main-class` along with `-classpath`  to the default command now defaults to `run` instead of `repl`:
+
+```
+▶ scala-cli --main-class Hello -classpath out
+Hello
+```
+
+* If the `run` sub-command is passed explicitly, it's sufficient to have a main class on the classpath (inputs aren't necessary then):
+
+```
+▶ scala-cli compile Hello.scala -d out
+▶ scala-cli run -classpath out 
+Hello
+```
+
+Added in [#1369](https://github.com/VirtusLab/scala-cli/pull/1369) by [@Gedochao](https://github.com/Gedochao)
+
+## Debugging for `run` and `test` sub-commands
+
+It is now possible to debug code ran by `run` and `test` sub-commands:
+
+```
+▶ scala-cli Main.scala --debug
+Listening for transport dt_socket at address: 5005
+Hello
+```
+
+This addresses [#1212](https://github.com/VirtusLab/scala-cli/issues/1212)
+
+Added in [#1389](https://github.com/VirtusLab/scala-cli/pull/1389) by [@wleczny](https://github.com/wleczny)
+
+## `--platform` option
+
+This option can be used to choose the platform, which should be used to compile and run application.
```

Comment:
```suggestion
This option can be used to choose the platform, which should be used to compile and run the application.
```

Reply:
might also want to mention that `--platform js` is an alias for `--js` and `--platform native` for `--native`.